src/modules/login/actions/login.js

```diff
@@ -3,6 +3,21 @@ import {browserHistory} from 'react-router';
 import { Server, Notification } from '../../core/helpers';
 
 const authService = Server.service('/auth');
+const schoolService = Server.service('/schools');
+const systemService = Server.service('/systems');
+
+String.prototype.capitalizeFirstLetter = function () {
```

Comment:
Whoa maybe don't extend the String class here but just make it either make it a normal function or put it somewhere so it's available for everything.